shell/client/transfers-client.js

```diff
@@ -164,8 +164,7 @@ Template.transfers.events({
     }
 
     if (!confirm(
-        "WARNING: You are giving " + destination + " permission to download all your " +
-        "grains from this server. Are you sure?")) {
+      TAPi18n.__("grains.grainlist.sandstormGrainListPage.transfers.warningGiving") + " " + destination + " " + TAPi18n.__("grains.grainlist.sandstormGrainListPage.transfers.warningPermission"))) {
```

Comment:
It seems like it would make a lot of sense to use parameters here, rather than concatenate strings, so that the translation string doesn't need to be split. Similarly for some of the other translations you added. What do you think?

Reply:
Well, yes, but how?

Reply:
There are lots of translations that have `%s` in them... try grepping for them to see how the parameter is passed in.

I think you can just add the params to the call, like `TAPi18n.__("translationName", param1, param2)`.

Reply:
I changed this to use parameters.